transit_odp/timetables/extract.py

```diff
@@ -120,6 +122,11 @@ def extract(self) -> ExtractedData:
             logger.debug("Extracting operating_profiles")
             operating_profiles = self.extract_operating_profiles()
             logger.debug("Finished extracting operating_profiles")
+
+            # Extract Lines
+            logger.debug("Extracting Lines")
+            lines = self.extract_lines()
```

Comment:
Can lines be extracted as part of service and then in transform stage, extract lines from service dataframe?